manifests/service.pp

```diff
@@ -1,6 +1,7 @@
 class mcollective::service {
-  service{$mcollective::service_name:
+  service{ 'mcollective':
```

Comment:
hmm, do we need to change this? Generally I believe the code will interact with `Class[mcollective::service]` so this wont matter.  Or did you find something not working without this change?

Reply:
Well, it failed here because I had a legacy setup… And somewhere had some legacy dependency `-> Service['mcollective']` that would have been changed to `-> Service[$mcollective::service_name]` from a totally different scope, which is awful.

`Class[mcollective::service]` is definitively the way to go.  I can drop this form the PR if you prefer to keep this that way.

Reply:
yeah lets drop it, keep the changes minimal, for sure the intention is a new OS needs only the yaml updates